tst/EnergyPlus/unit/Fixtures/EnergyPlusFixture.cc

```diff
@@ -132,6 +130,9 @@ void EnergyPlusFixture::TearDown()
         ObjexxFCL::gio::close(DataGlobals::OutputDElightIn, flags);
         ObjexxFCL::gio::close(DataGlobals::OutputFileShadingFrac, flags);
     }
+
+    clearAllStates();
```

Comment:
`clearAllStates` needs to be after the file closing, otherwise, the unit numbers are all zero and they don't really get closed, and the unit numbers keep ratcheting up into the 500s.

Reply:
This is a huge find.  Great job, detective @mjwitte !